config.js

```diff
@@ -76,7 +76,8 @@ var config = {
     subservice: '/gardens',
     providerUrl: 'http://192.168.56.1:4041',
     deviceRegistrationDuration: 'P1M',
-    defaultType: 'Thing'
+    defaultType: 'Thing',
+    appendMode: true
```

Comment:
I understand that if a parameter is not in config.js is an implicit `false`, isn't it?

Reply:
Yes. Internal behavior now has been changed in that sense

Reply:
NTC